src/components/pages/Offer/OfferDetails/OfferPlaceholder/OfferPlaceholder.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { ReactComponent as AddMediation } from './assets/add-mediation.svg'
+
+const OfferPlaceholder = () => (
+  <div>
+    <div className="of-placeholder">
+      <AddMediation />
+      {'Ajouter une image'}
+    </div>
+  </div>
```

Comment:
```suggestion
    <div className="of-placeholder">
      <AddMediation />
      {'Ajouter une image'}
    </div>
```
a deplacer dans OfferDetail